static/src/stylesheets/module/identity/_switches.scss

```diff
@@ -37,6 +37,7 @@ $checkbox-size: $gs-gutter / 1.25;
         @include mq(desktop) {
             grid-template-columns: repeat(2, 1fr);
             grid-auto-rows: 1fr;
+            width: gs-span(8);
```

Comment:
this component should not know about with at this scope (that way it can be added to any pages of any width). It will however take the full width of the page by default!